src/app-layout/drawer/interfaces.ts

```diff
@@ -75,7 +75,8 @@ export interface DrawerItem {
   };
   ariaLabels: DrawerItemAriaLabels;
   resizable?: boolean;
-  size?: number;
+  initialSize?: number;
```

Comment:
In other components we use `defaultSomething` to provide state only for the first render